internal/proto/query_coord.proto

```diff
@@ -313,8 +314,11 @@ message HandoffSegmentsRequest {
 
 message LoadBalanceRequest {
   common.MsgBase base = 1;
-  repeated int64 source_nodeIDs = 2;
+  repeated int64 source_nodeIDs = 2; // deprecated
   TriggerCondition balance_reason = 3;
+  int64 src_nodeID = 6;
+  int64 dst_nodeID = 5;
+  repeated int64 sealed_segmentIDs = 4;
```

Comment:
Is it better to change repeated int64 sealed_segmentIDs to repeated SegmentInfo sealed_segments = 4?